### ISSUE TITLE
Lodestone compass inventory sync bug between servers

### ISSUE THREAD

**Issue by jerro1105** [SpigotMC/BungeeCord#2922]
Players are able to copy items using this bug when travel between servers.
More details in this video
https://www.youtube.com/watch?v=KFm0cimwg54

**Comment by Mystiflow** [comment]
Are the items actually tangible? Can they be used on the new server? I'm pretty sure this would be a Lodestone plugin bug anyway?

**Comment by jerro1105** [comment]
Yes, the shulker box can actually be placed on the ground and items can be taken

**Comment by Janmm14** [comment]
the culprit of such a bug cannot be bungeecord. it might at most be easier to achieve this by using bungeecord.

This is an issue of your inventory synchronisation plugin.

**Comment by jerro1105** [comment]
Ah sorry, I forgot that it's inventory sync plugin's problem, I should report it to plugin author.
Also thanks for answering.
